packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -36,6 +37,20 @@ const DesignPreviewImage: React.FC< DesignPreviewImageProps > = ( {
 } ) => {
 	const isMobile = useViewportMatch( 'small', '<' );
 
+	if ( design.is_externally_managed && design.screenshot ) {
+		const fit = '479,360';
```

Comment:
Nit: Where do these numbers come from? Will these need to be updated in the future?

Currently the image fits nicely for all breakpoints:
| Breakpoint | Page |
|--------|--------|
| <320px | <img width="321" alt="CleanShot 2023-08-21 at 11 45 20@2x" src="https://github.com/Automattic/wp-calypso/assets/11555574/088a44c3-8965-459d-bf06-f039917fe189"> |
| <768px | <img width="770" alt="CleanShot 2023-08-21 at 11 45 37@2x" src="https://github.com/Automattic/wp-calypso/assets/11555574/cee395d8-adfb-40b1-8f61-ce3e0977ae6c"> |
| <1024px | <img width="1024" alt="CleanShot 2023-08-21 at 11 46 04@2x" src="https://github.com/Automattic/wp-calypso/assets/11555574/99018ee3-f7fc-4aa6-b28e-867a59bdf08a"> |

Reply:
Good point, I brought it from here where we also display the screenshot: https://github.com/Automattic/wp-calypso/blob/trunk/client/components/theme/index.jsx#L176

I didn't invest much in figuring out a way to share this value because it comes from `client/components` and this is a separate package so it must have all the info by itself or be passed by param. In this case, I decided to just replicate the code.

Reply:
Thanks for the context. It's good to know that the thumbnails are also a fixed aspect ratio. I wondered what was the original motivation behind moving the design picker to a separate package as it seems to cause some friction when we make updates. I've only found this point about the i18n wrapper: https://github.com/Automattic/wp-calypso/pull/51226#discussion_r597764782

Reply:
>  I wondered what was the original motivation behind moving the design picker to a separate package

I already asked me the same thing, but I didn't find an answer yet.